libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -513,6 +514,9 @@ userdata AP_MotorsMatrix_Scripting_Dynamic::factor_table field throttle'array AP
 include AP_InertialSensor/AP_InertialSensor.h
 singleton AP_InertialSensor rename ins
 singleton AP_InertialSensor method get_temperature float uint8_t 0 INS_MAX_INSTANCES
+singleton AP_InertialSensor method healthy boolean
```

Comment:
This function is just:
```
    bool healthy(void) const { return get_gyro_health() && get_accel_health(); }
```
So there is no need for the extra binding, you can do the same in lua as you have the other two bindings.

Reply:
Ok, i added it for completion, but i'll remove it if you think that's best :)

Reply:
This has been removed :)